src/core/packages/saved-objects/api-server-internal/src/lib/repository.security_extension.test.ts

```diff
@@ -845,14 +846,108 @@ describe('SavedObjectsRepository Security Extension', () => {
       });
     });
 
+    test(`returns empty authorization map for partially authorized if the authorized types are not part of the query`, async () => {
```

Comment:
Nit - the wording was confusing to me, but it could just be me
```suggestion
    test(`passes an empty type map to the query when partially authorized but no types are authorized for find or are hidden`, async () => {
```